metadata.yaml

```diff
@@ -0,0 +1,27 @@
+name: sdcore-gnbsim
+
+display-name: SD-CORE 5G GNBSIM
+summary: A Charmed Operator for SDCORE's GNBSIM component.
```

Comment:
Nitpicking, but I believe we should strive for consistency in the `SDCORE`/`SD-CORE` choice. That's my bad as well because we have a bit of a mix around projects.